projects/mui-angular/button/button.component.ts

```diff
@@ -1,36 +1,82 @@
-import { Component, ElementRef, Input, OnInit, ViewChild } from '@angular/core';
-
+import { Component, ElementRef, Input, ViewChild, HostListener, Renderer2 } from '@angular/core';
 
 @Component({
   selector: 'mui-button',
   template: `
-    <button class="mui-btn" #button>
+    <button class="mui-btn" #button [disabled]="disabled">
       <ng-content></ng-content>
+      <span class="mui-btn__ripple-container" #container><span class="mui-ripple" #ripple></span></span>
```

Comment:
Is the `#container` reference being used?

Reply:
No, will remove.